src/core/server/saved_objects/migrationsv2/integration_tests/actions.test.ts

```diff
@@ -100,6 +100,7 @@ describe('migration actions', () => {
 
   describe('fetchIndices', () => {
     it('resolves right empty record if no indices were found', async () => {
+      expect.assertions(1);
       const task = fetchIndices(client, ['no_such_index']);
       return expect(task()).resolves.toMatchInlineSnapshot(`
                 Object {
```

Comment:
I would just use `await` instead of returning the promise here. That's way more error-prone when adding new assertions imho (and allow to remove the `expect.assertions(1))

```ts
 const task = fetchIndices(client, ['no_such_index']);
 await expect(task()).resolves.toMatchInlineSnapshot(...)
```